python/llm/example/CPU/QLoRA-FineTuning/alpaca-qlora/alpaca_qlora_finetuning_cpu.py

```diff
@@ -52,6 +52,21 @@
 # import them from bigdl.llm.transformers.qlora to get a BigDL-LLM compatible Peft model
 from bigdl.llm.transformers.qlora import get_peft_model, prepare_model_for_kbit_training
 
+def get_int_from_env(env_keys, default):
+    """Returns the first positive env value found in the `env_keys` list or the default."""
+    for e in env_keys:
+        val = int(os.environ.get(e, -1))
+        if val >= 0:
+            return val
+    return default
+
+local_rank = get_int_from_env(["LOCAL_RANK","MPI_LOCALRANKID"], "0")
```

Comment:
PMI_RANK like [here](https://github.com/intel-analytics/BigDL/blob/main/docker/llm/finetune/lora/cpu/docker/lora_finetune.py#L111)? And pls test this PR before we merge it

Reply:
I have validate this pr in one node two sockets, do you want to test it in docker container?